### PR TITLE
Remove 'production entities enabled' feature toggle from the Service entity

### PR DESCRIPTION
**Pull request by MKodde**
The 'production entities enabled' feature toggle is not used by the PO's of SPD. They always allow the services to create production entites. Having this featue is illogical, and it can be cleaned up.

Note that a Doctrine migration was created to remove the column from the `service` table.

https://www.pivotaltracker.com/story/show/187053888

### CODE REVIEW

File: src/Surfnet/ServiceProviderDashboard/Application/Command/Service/EditServiceCommand.php [hunk 1]
```diff
@@ -51,7 +50,7 @@ public function __construct(
         private ?string $intakeStatus,
         private ?string $contractSigned,
         private ?string $surfconextRepresentativeApproved,
-        private ?string $privacyQuestionsAnswered,
+        private bool $privacyQuestionsAnswered,
```
Comment (parijke) [inline src/Surfnet/ServiceProviderDashboard/Application/Command/Service/EditServiceCommand.php:53]: not nullable anymore?

BTW, I heard a statement, that bool variables "sh|c"ould be named with `is` in front, like $isPrivacyQuestionAnswered.

Butt hat is probably not for now

Comment (parijke) [reply src/Surfnet/ServiceProviderDashboard/Application/Command/Service/EditServiceCommand.php:53]: If is is not nullebale, maybe initilize it with false to be sure?

File: src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php [hunk 2]
```diff
@@ -21,12 +21,9 @@
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
 use Doctrine\ORM\Mapping as ORM;
-use Surfnet\ServiceProviderDashboard\Domain\Entity\PrivacyQuestions;
```
Comment (parijke) [inline src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php:24]: Not sure why this is gone, was it obsolete already?

**Review by parijke — APPROVED** [review]
Small suggestions / questions, no blockers to merge IMHO

File: src/Surfnet/ServiceProviderDashboard/Application/Command/Service/EditServiceCommand.php (hunk 1 above)
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Application/Command/Service/EditServiceCommand.php:53]: I looked it up, and it should always be set. My own testing and the CI functional tests did confirm my assumption. I feel confident to leave it like this.

File: src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php (hunk 2 above)
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php:24]: That was a useless import statement, the PrivacyQuestions are in the same namespace. So importing them is not usefull.

**Review by MKodde — COMMENTED** [review]
Thanks for your review @parijke

File: src/Surfnet/ServiceProviderDashboard/Application/Command/Service/EditServiceCommand.php (hunk 1 above)
Comment (parijke) [reply src/Surfnet/ServiceProviderDashboard/Application/Command/Service/EditServiceCommand.php:53]: https://3v4l.org/gWhSF

**Pull request merged**